decidim-core/lib/decidim/permissions_registry.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Decidim
+  #
+  # Takes care of holding and accessing permissions classes for each artifact.
+  #
+  class PermissionsRegistry
+    def initialize
+      @registry = {}
+    end
+
+    # Syntactic sugar for the `chain_for` instance method.
+    def self.chain_for(artifact)
+      ::Decidim.permissions_registry.chain_for(artifact)
```

Comment:
Should this be calling the instance method? I think this method is calling itself

Reply:
Hi,
No, the reviewed method is called via `PermissionsRegistry.chain_for(artifact)`, then what is inside the method may be rewritten:
```ruby
    # Syntactic sugar for the `chain_for` instance method.
    def self.chain_for(artifact)
      registry= ::Decidim.permissions_registry
      registry.chain_for(artifact)
    end
```

Reply:
Ah yes, you're absolutely right, I misread `::Decidim.permissions_registry` as `::Decidim::PermissionsRegistry`, my bad!